src/Apps/Search/routes.tsx

```diff
@@ -32,6 +35,23 @@ export const routes: RouteConfig[] = [
     children: [
       {
         path: "/",
+        render: ({ props, Component }) => {
+          if (!props || !Component) {
+            return null
+          }
+
+          return (
+            <Provider
```

Comment:
Why not keep this within the component?

Reply:
Ah, good catch! This was something i was moving around when debugging the radio button stuff. Will revert.